### PR TITLE
[stable22] let user choose notification email in user settings

### PR DESCRIPTION
**Pull request by blizzz**
adjusted backport of #28840

### CODE REVIEW

**Review by Pytal — APPROVED** [review]
🚀🚀

**Comment by blizzz** [comment]
failing tests unrelated

File: apps/settings/src/components/PersonalInfo/EmailSection/Email.vue [hunk 1]
```diff
@@ -239,6 +290,19 @@ export default {
 			}
 		},
 
+	  handleSetNotificationMailResponse({ notificationEmail, status, errorMessage, error }) {
+		  if (status === 'ok') {
+				this.$emit('update:notification-email', notificationEmail)
```
Comment (PVince81) [inline apps/settings/src/components/PersonalInfo/EmailSection/Email.vue:295]: oops indent

**Review by PVince81 — APPROVED** [review]
:+1:

File: apps/settings/src/components/PersonalInfo/EmailSection/Email.vue (hunk 1 above)
Comment (blizzz) [reply apps/settings/src/components/PersonalInfo/EmailSection/Email.vue:295]: indeed. idk, but the indentation is almost always fucked up in phpstorm and i need to fix it manually. done now.

**Pull request merged**